### ISSUE TITLE
(docs) add examples on how to use autoevals.llm 

### ISSUE THREAD

**Issue by ishaan-jaff** [braintrustdata/autoevals#30]
It's unclear how to use the different types of autoevals, can you add example usage and supported input params 

![Screenshot 2023-11-09 at 4 02 55 PM](https://github.com/braintrustdata/autoevals/assets/29436595/da3b9e54-571c-4315-b827-cbaa2366393a)

**Comment by ishaan-jaff** [comment]
also showing expected output would be useful

**Comment by ankrgyl** [comment]
Thanks for the idea! We definitely have some work to do on the docs here. Would you be interested in contributing? Happy to work through a few ideas on how to do it in an automated way.

**Comment by ishaan-jaff** [comment]
yeah open to helping here - I actually just added docs on how to use auto evals with litellm and wanted to add more details there: https://docs.litellm.ai/docs/tutorials/eval_suites#using-litellm-with-autoeval

**Comment by ibolmo** [comment]
@ishaan-jaff 

I made some improvements to our docs based on your suggestion. Here are the docs if you don't mind a review: https://www.braintrust.dev/docs/reference/autoevals/python

The TypeScript ones are pending, but I have another ticket to get that fixed. 

Please reopen the issue if you find anything else.
